services/accountmanager/dirk/service.go

```diff
@@ -497,3 +497,8 @@ func (s *Service) AccountByPublicKey(_ context.Context, pubkey phase0.BLSPubKey)
 	}
 	return account, nil
 }
+
+// HasSlashingProtection returns true if the account manage provides built-in slashing protection.
+func (*Service) HasSlashingProtection() bool {
+	return false
```

Comment:
Is this the correct way around? I thought dirk had built in slashing protection and the wallet didnt?

Reply:
Yep wrong way round; fixed.